docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -5,531 +5,129 @@
-
-.. warning::
-
-    Please note that some modules have additional requirements that need to be satisfied on the 'target' machine (the managed node). These requirements should be listed in the module documentation.
-
-.. note::
-
-   * If you have SELinux enabled on remote nodes, you will also want to install libselinux-python on them before using any copy/file/template related functions in Ansible. You can use the :ref:`yum module<yum_module>` or :ref:`dnf module<dnf_module>` in Ansible to install this package on remote systems that do not have it.
-
-   * By default, before the first Python module in a playbook runs on a host, Ansible attempts to discover a suitable Python interpreter on that host. You can override the discovery behavior by setting the :ref:`ansible_python_interpreter<ansible_python_interpreter>` inventory variable to a specific interpreter, and in other ways. See :ref:`interpreter_discovery` for details.
-
-   * Ansible's :ref:`raw module<raw_module>`, and the :ref:`script module<script_module>`, do not depend on a client side install of Python to run.  Technically, you can use Ansible to install a compatible version of Python using the :ref:`raw module<raw_module>`, which then allows you to use everything else. For example, if you need to bootstrap Python 2 onto a RHEL-based system, you can install it as follows:
+=========================
 
-     .. code-block:: shell
+For your control node (the machine that runs Ansible), you can use nearly any UNIX-like machine with Python 3.8 or newer installed. This includes Red Hat, Debian, Ubuntu, macOS, BSDs, and Windows under a `Windows Subsystem for Linux (WSL) distribution <https://docs.microsoft.com/en-us/windows/wsl/about>`_. Windows without WSL is not natively supported as a control node; see `Matt Davis' blog post <http://blog.rolpdog.com/2020/03/why-no-ansible-controller-for-windows.html>`_ for more information.
 
-        $ ansible myhost --become -m raw -a "yum install -y python2"
+.. _getting_ansible:
 
 .. _what_version:
 
-Selecting an Ansible artifact and version to install
+Selecting an Ansible package and version to install
 ====================================================
 
-Starting with version 2.10, Ansible distributes two artifacts: a community package called ``ansible`` and a minimalist language and runtime called ``ansible-core`` (called `ansible-base` in version 2.10). Choose the Ansible artifact and version that matches your particular needs.
-
-Installing the Ansible community package
-----------------------------------------
-
-The ``ansible`` package includes the Ansible language and runtime plus a range of community curated Collections. It recreates and expands on the functionality that was included in Ansible 2.9.
-
-You can choose any of the following ways to install the Ansible community package:
```

Comment:
I would add `You should look for ansible within the package manager for the distribution you are using, if you cannot find it or not find the specific version you want,  the following section shows how to install using Python specific package manager (pip).`

Reply:
I agree. If we're going to remove exhaustive instructions for various distributions, we should at least mention that such packages exist.

Reply:
We are purposefully steering people away from using the package manager, and towards pip as the only officially supported installation mechanism.  I did have something before that indicated that you _may_ find packages in your OS distribution package manager, and are free to use them, but with some caveat that we cannot guarantee anything about said distro packages.

I can probably figure a way to add that back.

Reply:
added